lib/grr.js

```diff
@@ -1639,6 +1656,8 @@ function stepReviewCRs(arg, cb) {
             arg.log.trace({err: pipelineErr}, 'pipelineErr');
```

Comment:
Can drop this log.trace given that the error is being passed back in `cb`, it will be shown in the caller of this (in our case in `mainFinish()`).

Reply:
Yep, thanks - this also landed via:

commit beca7c1e1e3c89382500e6c4487153ecdf855630
Author: Tim Foster <tim.foster@joyent.com>
Date:   Wed Aug 14 14:20:38 2019 +0100

    Trent's patch to add more spacing between diffs when showing reviews